libs/ui/inputs/src/lib/dropdown-selector/dropdown-selector.component.html

```diff
@@ -1,15 +1,15 @@
 <div class="flex flex-col items-start sm:flex-row sm:items-center relative">
   <label
     *ngIf="showTitle"
-    class="block tracking-wide text-sm mb-2 sm:mb-0 sm:mr-2 whitespace-no-wrap"
+    class="block tracking-wide text-sm mb-2 sm:mb-0 sm:mx-2 whitespace-no-wrap shrink-0"
     [attr.for]="id"
   >
     {{ title }}
   </label>
   <select
     [id]="id"
     (change)="this.selectValue.emit($event.target.value)"
-    class="text-main bg-white border border-gray-300 py-[10px] px-[16px] rounded-[0.25em] cursor-pointer leading-tight focus:outline-none hover:text-primary-darker hover:border-primary-lighter focus:ring-primary-lightest focus:ring-4"
+    class="w-full max-w-xs text-main bg-white border border-gray-300 py-[10px] px-[16px] rounded-[0.25em] cursor-pointer leading-tight focus:outline-none hover:text-primary-darker hover:border-primary-lighter focus:ring-primary-lightest focus:ring-4"
```

Comment:
Ok for `w-full` indeed it helps making the dropdown shrink when there's not enough room. But why add `max-w-xs`?